dialects/ydb.go

```diff
@@ -0,0 +1,1189 @@
+		"UNCONDITIONAL":     true,
+		"UNION":             true,
+		"UNIQUE":            true,
+		"UNKNOWN":           true,
+		"UPDATE":            true,
+		"UPSERT":            true,
+		"USE":               true,
+		"USER":              true,
+		"USING":             true,
+		"VACUUM":            true,
+		"VALUES":            true,
+		"VARIANT":           true,
+		"VIEW":              true,
+		"VIRTUAL":           true,
+		"WHEN":              true,
+		"WHERE":             true,
+		"WINDOW":            true,
+		"WITH":              true,
+		"WITHOUT":           true,
+		"WRAPPER":           true,
+		"XOR":               true,
+		"TRUE":              true,
+		"FALSE":             true,
+	}
+
+	ydbQuoter = schemas.Quoter{
+		Prefix:     '`',
+		Suffix:     '`',
+		IsReserved: schemas.AlwaysReserve,
+	}
+)
+
+const (
+	// numeric types
+	yql_Bool = "BOOL"
+
+	yql_Int8  = "INT8"
+	yql_Int16 = "INT16"
+	yql_Int32 = "INT32"
+	yql_Int64 = "INT64"
+
+	yql_Uint8  = "UINT8"
+	yql_Uint16 = "UINT16"
+	yql_Uint32 = "UINT32"
+	yql_Uint64 = "UINT64"
+
+	yql_Float   = "FLOAT"
+	yql_Double  = "DOUBLE"
+	yql_Decimal = "DECIMAL"
+
+	// string types
+	yql_String       = "STRING"
+	yql_Utf8         = "UTF8"
+	yql_Json         = "JSON"
+	yql_JsonDocument = "JSONDOCUMENT"
+	yql_Yson         = "YSON"
+
+	// Data and Time
+	yql_Date      = "DATE"
+	yql_DateTime  = "DATETIME"
+	yql_Timestamp = "TIMESTAMP"
+	yql_Interval  = "INTERVAL"
+
+	// Containers
+	yql_List = "LIST"
+)
+
+func toYQLDataType(t string, defaultLength, defaultLength2 int64) (yqlType string) {
+	switch v := t; v {
+	case schemas.Bool, schemas.Boolean:
+		yqlType = yql_Bool
+		return
+	case schemas.TinyInt:
+		yqlType = yql_Int8
+		return
+	case schemas.UnsignedTinyInt:
+		yqlType = yql_Uint8
+		return
+	case schemas.SmallInt:
+		yqlType = yql_Int16
+		return
+	case schemas.UnsignedSmallInt:
+		yqlType = yql_Uint16
+		return
+	case schemas.MediumInt:
+		yqlType = yql_Int32
+		return
+	case schemas.UnsignedMediumInt:
+		yqlType = yql_Uint32
+		return
+	case schemas.BigInt:
+		yqlType = yql_Int64
+		return
+	case schemas.UnsignedBigInt:
+		yqlType = yql_Uint64
+		return
+	case schemas.Int, schemas.Integer:
+		yqlType = yql_Int32
+		return
+	case schemas.UnsignedInt:
+		yqlType = yql_Uint32
+		return
+	case schemas.Float:
+		yqlType = yql_Float
+		return
+	case schemas.Double:
+		yqlType = yql_Double
+		return
+	case schemas.Blob:
+		yqlType = yql_String
+		return
+	case schemas.Json:
+		yqlType = yql_Json
+		return
+	case schemas.Array:
+		yqlType = yql_List
+		return
+	case schemas.Varchar, schemas.Text:
+		yqlType = yql_Utf8
+		return
+	case schemas.TimeStamp, schemas.DateTime:
+		yqlType = yql_Timestamp
+		return
+	case schemas.Interval:
+		yqlType = yql_Interval
+		return
+	default:
+		yqlType = yql_String
+	}
+	return
+}
+
+func yqlToSQLType(yqlType string) (sqlType schemas.SQLType) {
+	switch yqlType {
+	case yql_Bool:
+		sqlType = schemas.SQLType{Name: schemas.Bool, DefaultLength: 0, DefaultLength2: 0}
```

Comment:
replace to `return  schemas.SQLType{Name: schemas.Bool, DefaultLength: 0, DefaultLength2: 0}`

Reply:
in all cases